test-data/unit/check-class-namedtuple.test

```diff
@@ -464,7 +427,7 @@ UserDefined(1)  # E: Argument 1 to "UserDefined" has incompatible type "int"; ex
 [builtins fixtures/list.pyi]
 
 [case testNewNamedTupleWithDefaultsStrictOptional]
-# flags: --strict-optional --python-version 3.6
+# flags: --strict-optional
```

Comment:
I thought `--strict-optional` was the default in our tests now:

```suggestion
```

Reply:
There are lots of `--strict-optional` markers in the code base:

```
» ag -Q 'strict-optional' . --stats-only
355 matches
62 files contained matches
1319 files searched
13443951 bytes searched
0.026860 seconds
                   
```

Let's not touch it right now.